### PR TITLE
BAH-3069 | To display link ABHA Address button based on configuration

### PR DESCRIPTION
**Pull request by SanoferSameera**

### CODE REVIEW

File: src/api/constants.js [hunk 1]
```diff
@@ -39,6 +39,7 @@ export const globalPropertyUrl = "/openmrs/ws/rest/v1/bahmnicore/sql/globalprope
 export const cmSuffixProperty = 'bahmniHip.CM_SUFFIX'
 export const enableHealthIdVerification = 'bahmniHip.enableHealthIdVerification'
 export const enableDemographics = 'bahmniHip.EnableDemographics'
+export const enableLinkABHAAddress = 'bahmniHip.EnableLinkABHAAddress'
```
Comment (angshu) [inline src/api/constants.js:42]: if you change the other PR for global property name, remember to change this as well.

File: src/components/creation/LinkABHAAddress.jsx [hunk 2]
```diff
@@ -27,7 +29,16 @@ const LinkABHAAddress = (props) => {
         setProceed(true);
     }
 
+    function fetchLinkAddressGlobalPropery(){
```
Comment (kiranmairavipati1978) [inline src/components/creation/LinkABHAAddress.jsx:32]: minor: typo t is missing in property `fetchLinkAddressGlobalPropery`

Comment (kiranmairavipati1978) [reply src/components/creation/LinkABHAAddress.jsx:32]: Also there are no tests for the components in this repo. We can create tech debt card for the same.

Comment (SanoferSameera) [reply src/components/creation/LinkABHAAddress.jsx:32]: yeah. I'll fix the typo. Thanks.

**Pull request merged**